client/js/entities/model/bioticState.js

```diff
@@ -28,9 +28,7 @@ return function BioticState(entityState, health) {
 
 	self.prettyCloseTo = function (other) {
 		return closeVec(self.entityState.body.pos, other.entityState.body.pos)
-			&& closeVec(self.entityState.body.dir, other.entityState.body.dir)
-			&& closeVec(self.entityState.body.vel, other.entityState.body.vel)
-			&& close(self.health.life, other.health.life);
+			&& closeVec(self.entityState.body.vel, other.entityState.body.vel);
```

Comment:
This doesn't seem correct. Our directions in the entityState should match, right?